src/Amazon.php

```diff
@@ -134,6 +149,10 @@ public static function get_available_shipping_methods(?\WC_Order $order) {
       self::$firstAvailableMethod[$order->get_id()] = reset($available_methods[0]['rates']);
     }
 
+    // Clean up
```

Comment:
I am considering also emptying the cart as part of our clean up.

Reply:
Yeah, emptying the cart would be good to avoid it from starting another session on shutdown.